service_crategen/src/commands/generate/codegen/rest_xml.rs

```diff
@@ -159,7 +160,8 @@ impl GenerateProtocol for RestXmlGenerator {
                              shape: &Shape,
                              service: &Service)
                              -> Option<String> {
-        Some(xml_payload_parser::generate_deserializer(name, shape, service))
+        let ty = get_rust_type(service, name, shape, self.timestamp_type());
```

Comment:
Only thing I have is a nitpick with the variable name `ty`. Could be changed to be clearer.  `rust_type`?